dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityAdminMode.java

```diff
@@ -13,9 +14,9 @@
  */
 public class VelocityAdminMode extends VelocityLiveMode {
 
-    public VelocityAdminMode(final HttpServletRequest request, final HttpServletResponse response, final String uri,
+    public VelocityAdminMode(final HttpServletRequest request, final HttpServletResponse response, final IHTMLPage htmlPage,
```

Comment:
Shouldn't we keep the former constructor for backward compatibility and deprecate it if necessary?

Reply:
agree

Reply:
100% agreed. Specially true if a Hotfix Plugin needs to be created for Support.

Reply:
done https://github.com/dotCMS/core/pull/17859/commits/ec404552260c00230d3cce3ac019d69c6a6cea57